taskt/Core/ExtensionMethods.cs

```diff
@@ -67,15 +67,15 @@ public static string ConvertToUserVariable(this String str, object sender)
                         {
                             //get the value from the list
 
-                            complexJsonVariable = complexJsonVariable.Replace("^" + potentialSubVariable + "^", matchingVar.GetDisplayValue());
-                            continue;
-                        }
-
-                    }
-
-
-
-                    //split by json select token pointer
+                            complexJsonVariable = complexJsonVariable.Replace("^" + potentialSubVariable + "^", matchingVar.GetDisplayValue());
+                            continue;
+                        }
+
+                    }
+
+
+
+                    //split by json select token pointer
```

Comment:
Why does this file have so many lines being changed, but not being edited? If possible could you exclude these changes somehow, or find a way to prevent them in the future? I think you might be changing the line endings.